gdown/download.py

```diff
@@ -188,11 +197,11 @@ def download(
         f = output
 
     try:
-        total = res.headers.get("Content-Length")
-        if total is not None:
-            total = int(total)
+        content_length = res.headers.get("Content-Length")
+        if content_length is not None:
+            content_length = int(content_length)
```

Comment:
This rename was accidental. 
Originally I was planning to use content length to determine if a file is fully downloaded. But it seems Google is not sending content length in `Content-Length` header or `Content-Disposition` header.  
I forgot to rename it back to the original variable name `total`.  
I'll rename and push again if needed.